package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-google-cloudfunctions",
-  "version": "2.4.2",
+  "version": "3.0.0",
```

Comment:
This should done with dedicated release PR. Please get familiar with https://github.com/serverless/serverless-google-cloudfunctions/blob/master/RELEASE_PROCESS.md